### PR TITLE
font-family-added

### PR DESCRIPTION
**Pull request by bharatgupta77**
font-family added to some pages to make it similar to home page.

### CODE REVIEW

File: css/pages/coc.css [hunk 1]
```diff
@@ -4,6 +4,10 @@
   font-weight: 550;
   padding: 20px;
  }
+
+ body {
+  font-family: 'Roboto', sans-serif;
+}
```
Comment (ankushdharkar) [inline css/pages/coc.css:10]: Doesn't this come from `main.css`?

File: css/pages/discord.css [hunk 2]
```diff
@@ -1,3 +1,8 @@
+
+body {
+  font-family: 'Roboto', sans-serif;
+}
+
```
Comment (ankushdharkar) [inline css/pages/discord.css:5]: Re: main.css?

**Comment by bharatgupta77** [comment]
Yeah ankush, I haven't seen that. I just had to add link in html pages.  Sorry my bad.

File: css/pages/coc.css (hunk 1 above)
Comment (bharatgupta77) [reply css/pages/coc.css:10]: Yeah ankush, I haven't seen that. I just had to add link in html pages. Sorry my bad. Will remove it in next pr.

**Pull request merged**